config/variable_binning.py

```diff
@@ -38,14 +44,14 @@
 # 'lepTopRap' : [-3.0, -1.0979999999999999, -0.438, 0.0, 0.438, 1.0979999999999999, 3.0],
 # 'ttbarRap' : [-3.0, -1.3679999999999999, -0.93, -0.54, -0.195, 0.0, 0.195, 0.54, 0.93, 1.3679999999999999, 3.0],
 
-'MET' : [0.0, 48.0, 86.0, 174.0, 702.0],
-'WPT' : [0.0, 56.0, 94.0, 135.0, 228.0, 786.0],
-'NJets' : [3.5, 4.5, 5.5, 9.5, 20.5],
-'HT' : [100.0, 237.0, 297.0, 382.0, 592.0, 1852.0],
-'ST' : [130.0, 353.0, 426.0, 529.0, 773.0, 2237.0],
-'lepton_pt' : [30.0, 40.0, 52.0, 71.0, 124.0, 442.0],
-# # 'lepton_eta' : [-2.5, -0.7, 0, 0.7, 2.5],
-# 'abs_lepton_eta' : [0, 0.7, 2.5],
+'MET' : [0.0, 51.0, 97.0, 281.0],
+'WPT' : [0.0, 57.0, 99.0, 146.0, 334.0],
+'NJets' : [3.5, 4.5, 5.5, 9.5],
+'HT' : [100.0, 243.0, 300.0, 375.0, 675.0],
+'ST' : [123.0, 360.0, 431.0, 524.0, 896.0],
+'lepton_pt' : [23.0, 39.0, 50.0, 66.0, 130.0],
+'abs_lepton_eta' : [0.0, 0.28800000000000003, 0.588, 0.927, 2.2830000000000004],
```

Comment:
I think this is a good candidate for the consolidation of bins
```python
-> [0.0, 0.3, 0.6, 0.9, 2.1]
```
The last bins will depend on the cut in the phase space in question. How far do we want to extrapolate?